CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 
 ## Version 2.6.0-beta2
+- [Implement unhandled exception auto-tracking (500 requests) for MVC 5 and WebAPI 2 applications] (https://github.com/Microsoft/ApplicationInsights-dotnet-server/pull/847)
 - [Added a max length restriction to values passed in through requests.](https://github.com/Microsoft/ApplicationInsights-dotnet-server/pull/810)
```

Comment:
Please add your comment to "2.6.0-beta3". Beta 2 is being released this week. I can explain offline.